src/components/Login/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FormEvent, useState } from 'react';
+import styles from './Login.module.css';
+
+export default function Login(): React.ReactElement {
+    const [email, setEmail] = useState<string | undefined>();
+    const [password, setPassword] = useState<string | undefined>();
+
+    function validateForm() {
+        return email && email.length > 0 && password && password.length > 0;
```

Comment:
I think this should be enough, because empty string and undefined will be falsy
```suggestion
        return email && password;
```

Reply:
wanted to make sure, but when I think about it, this is probably better.  Will make a change.